### PR TITLE
fix(curriculum): correct link text

### PR DESCRIPTION
**Pull request by alirezaghey**
The correct term is assignment operator.

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #XXXXX

### CODE REVIEW

**Review by gikf — DISMISSED** [review]
LGTM :+1:

**Review by Nirajn2311 — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/generate-random-fractions-with-javascript.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Random numbers are useful for creating random behavior.
 
 JavaScript has a `Math.random()` function that generates a random decimal number between `0` (inclusive) and not quite up to `1` (exclusive). Thus `Math.random()` can return a `0` but never quite return a `1`
```
[the remark this answers is not in the document]
Comment (alirezaghey) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/generate-random-fractions-with-javascript.md:14]: I agree. What do I need to do? Click on "Add suggestion to batch" or something else?

[the remark this answers is not in the document]
Comment (Nirajn2311) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/generate-random-fractions-with-javascript.md:14]: You can click the `Commit suggestion` button to add in the suggestion. Only thing to change will be the commit message.

**Review by Nirajn2311 — APPROVED** [review]
LGTM 👍

**Review by Sembauke — APPROVED** [review]
LGTM 👍

**Pull request merged**